app/javascript/application.js

```diff
@@ -1,8 +1,8 @@
 /* global window */
 
+import './jQueryGlobalizer.js'
```

Comment:
why the order change here?

Reply:
I changed the order because Bootstrap depends on jquery. Loading Jquery first, so other bootstrap features work correctly. This also fixes some console errors.